Source/Core/Core/IPC_HLE/WII_IPC_HLE_Device_sdio_slot0.cpp

```diff
@@ -63,19 +63,28 @@ void CWII_IPC_HLE_Device_sdio_slot0::OpenInternal()
 {
 	const std::string filename = File::GetUserPath(D_WIIROOT_IDX) + "/sd.raw";
 	m_Card.Open(filename, "r+b");
+	if (m_Card)
+		return;
+	
+	WARN_LOG(WII_IPC_SD, "Failed to open SD Card image, trying read-only access...");
+	m_Card.Open(filename, "rb");
+	if (m_Card)
+	{
+		SConfig::GetInstance().bEnableMemcardSdWriting = false;
```

Comment:
Why should Dolphin open all SD cards as read-only because GC memory cards open their files repeatedly? The two are not related.